kahuna/public/js/components/gr-permissions-filter/gr-permissions-filter.tsx

```diff
@@ -85,15 +96,19 @@ const PermissionsFilter: React.FC<PermissionsWrapperProps> = ({ props }) => {
     }
   };
 
-  const handleQueryChange = (e: any ) => {
-    const newQuery = e.detail.query ? (" " + e.detail.query) : "";
+  const handleLogoClick = (event: LogoClickEvent) => {
+    setLogoClick({logoClick: true, showPaid: event.detail.showPaid});
+  };
```

Comment:
I don't follow the need for this logoClick state, as far as I can tell it's only used to trigger the useEffect block below, which only sets the isChargeable state. I'm pretty sure you can replace all of that by directly setting the isChargeable state

```suggestion
  const handleLogoClick = (event: LogoClickEvent) => {
    setIsChargeable(event.detail.showPaid);
  };
```

Reply:
Have simplified code inline with suggestion